recipes/cmdline_provenance/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "cmdline_provenance" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "7abbffd72676c8bed10713790fb46cef44538d709085d6e1e7007d089f4b030d"
```

Comment:
You shouldn't have quotes around the sha, because it messes up the auto-tick bot currently.

Reply:
thanks for picking that up!